scripts/sct_pipeline_makefig.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+#
+# Generate figure from the output of sct_pipeline.
+#
+# Copyright (c) 2013 Polytechnique Montreal <www.neuro.polymtl.ca>
+# Author: Julien Cohen-Adad
+# Created: 2018-05-21
+# License: https://github.com/neuropoly/spinalcordtoolbox/blob/master/LICENSE
+
+# TODO: generalize this code for other things than Dice
+
+import argparse
+
+
+def get_parser():
+    parser = argparse.ArgumentParser(
+        description='Generate figure from the output of sct_pipeline.')
+    parser.add_argument("-i",
+                        help="Pickle file(s) generated by sct_pipeline. You can input several files, separated by space.",
+                        nargs='+',
+                        required=True)
+    parser.add_argument("-label",
+                        help="Label name for each pickle data.",
+                        nargs='+',
+                        required=True)
+    parser.add_argument("-v",
+                        help="Verbose: 0 = no verbosity, 1 = verbose (default).",
+                        choices=('0', '1'),
+                        type=int,
+                        default=1)
+    return parser
+
+
+def main(args):
+    import sct_utils as sct
+    import pickle
+    import numpy as np
+    import matplotlib.pyplot as plt
+
+    sct.start_stream_logger()
+
+    # fs = 10  # font size
+    nb_plots = args.i.__len__()
+
+    list_data = []
+    text_results = []  # numerical results to display inside the figure
+    for fname_pickle in args.i:
+        df = pickle.load(open(fname_pickle, "rb"))
```

Comment:
Could be changed to `io.open()` (more explanatory IMO, but it's a question of taste/habits)

Reply:
fixed in 9d77c82139345e4600a708702d8ba06729c7a1f6